superset/migrations/versions/cf50994ebd3a_tables_unique_on_table_name_and_database.py

```diff
@@ -0,0 +1,72 @@
+"""tables unique on table name and database
```

Comment:
@frankfarrell why is this migration required, given that it seems you're only querying the data.

Reply:
@john-bodley The query updates are to match the table uniqueness constraint correctly.
This migration is to put that constraint in place correctly for sqlite, the previous migrations don't manage to drop the old constraint